js/src/lib/collectors/span/mod.ts

```diff
@@ -36,11 +31,10 @@ export class SpanCollector implements Collector {
   }
 
   public async setNames(wasm: ArrayBuffer | WebAssembly.Module) {
```

Comment:
Did we want to keep `WebAssembly.Module` as part of the actual signature here? I remember you added it to support testing the JS integration. If we do keep it, I wonder if it should go anywhere else too?

Reply:
Yeah it was intentional.  I pushed up a commit with a new type `WASM`, to make this cleaner.  Also updated the `start` signature of the Adapters to take this new type